bigbluebutton-html5/imports/ui/components/whiteboard/shapes/helpers.js

```diff
@@ -0,0 +1,51 @@
+// applies zooming to the stroke thickness
+const zoomStroke = (thickness, widthRatio, heightRatio) => {
+  let ratio;
+
+  ratio = (widthRatio  + heightRatio) / 2;
+  return thickness * 100 / ratio;
+};
+
+const formatColor = (color) => {
+  //let color = this.props.shape.shape.shape.color;
+  if (color == null) {
```

Comment:
```
undefined == null returns true
undefined === null returns false
```
'==' just covers more cases in our case, generally we check if that value exists or not, that's why we have such checks all over the code.
What potentially bad scenarios do you see here?